migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc

```diff
@@ -35,10 +34,9 @@ include::modules/migration-rolling-back-migration-cli.adoc[leveloffset=+2]
 
 include::modules/migration-known-issues.adoc[leveloffset=+1]
 
+
```

Comment:
Is this an extraneous `+` that should be deleted?

Reply:
Yes. Thanks for catching it.